src/Message.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace ResellerClub;
+
+class Message
+{
+    /**
+     * @var string
+     */
+    protected $message;
```

Comment:
Can you change this to `private` please, it's always best to start as strict as possible and open up as and when.

Reply:
Sure, copied from similar in `Status`, but perhaps `Status` is extended

Reply:
Possibly, possibly an oversight when it was first coded.